docs/ksql-clickstream-demo/non-docker-clickstream.rst

```diff
@@ -0,0 +1,364 @@
+         CUSTOMER_CLICKSTREAM      | CUSTOMER_CLICKSTREAM      | JSON   
+         CLICKSTREAM               | clickstream               | JSON 
+
+13. From the the KSQL CLI, verify that data is being streamed through
+    various tables and streams.
+
+    **View clickstream data**
+
+    .. code:: bash
+
+        ksql> SELECT * FROM CLICKSTREAM LIMIT 5;
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        1503585407989 | 222.245.174.248 | 1503585407989 | 24/Aug/2017:07:36:47 -0700 | 233.90.225.227 | GET /site/login.html HTTP/1.1 | 407 | 19 | 4096 | Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
+        1503585407999 | 233.168.257.122 | 1503585407999 | 24/Aug/2017:07:36:47 -0700 | 233.173.215.103 | GET /site/user_status.html HTTP/1.1 | 200 | 15 | 14096 | Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
+        1503585408009 | 222.168.57.122 | 1503585408009 | 24/Aug/2017:07:36:48 -0700 | 111.249.79.93 | GET /images/track.png HTTP/1.1 | 406 | 22 | 4096 | Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
+        1503585408019 | 122.145.8.244 | 1503585408019 | 24/Aug/2017:07:36:48 -0700 | 122.249.79.233 | GET /site/user_status.html HTTP/1.1 | 404 | 6 | 4006 | Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
+        1503585408029 | 222.152.45.45 | 1503585408029 | 24/Aug/2017:07:36:48 -0700 | 222.249.79.93 | GET /images/track.png HTTP/1.1 | 200 | 29 | 14096 | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36
+        LIMIT reached for the partition.
+        Query terminated
+
+    **View the events per minute**
+
+    .. code:: bash
+
+        ksql> SELECT * FROM EVENTS_PER_MIN_TS LIMIT 5;
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        1503585450000 | 29^�8 | 1503585450000 | 29 | 19
+        1503585450000 | 37^�8 | 1503585450000 | 37 | 25
+        1503585450000 | 8^�8 | 1503585450000 | 8 | 35
+        1503585450000 | 36^�8 | 1503585450000 | 36 | 14
+        1503585450000 | 24^�8 | 1503585450000 | 24 | 22
+        LIMIT reached for the partition.
+        Query terminated
+
+    **View pages per minute**
+
+    .. code:: bash
+
+        ksql> SELECT * FROM PAGES_PER_MIN LIMIT 5;
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        1503585475000 | 4 : Window{start=1503585475000 end=-} | 4 | 14
+        1503585480000 | 25 : Window{start=1503585480000 end=-} | 25 | 9
+        1503585480000 | 16 : Window{start=1503585480000 end=-} | 16 | 6
+        1503585475000 | 25 : Window{start=1503585475000 end=-} | 25 | 20
+        1503585480000 | 37 : Window{start=1503585480000 end=-} | 37 | 6
+        LIMIT reached for the partition.
+        Query terminated    
+
+14. Go to your terminal and send the KSQL tables to Elasticsearch and
+    Grafana.
+
+    1. From your terminal, navigate to the demo directory:
+
+       ::
```

Comment:
Is this correct .rst syntax?  I think so, but the other instructions below use:

```
.. code:: bash
```
Might also be better to stick to `bash` because of proper syntax highlighting for these terminal instructions.

Reply:
yes, it renders correctly. this content is migrated from .md using pandoc - so for whatever reason, pandoc chose this syntax, in this case.